src/Symfony/Component/OptionsResolver/OptionsResolver.php

```diff
@@ -663,14 +663,16 @@ public function remove($optionNames)
         }
 
         foreach ((array) $optionNames as $option) {
-            unset($this->defined[$option]);
-            unset($this->defaults[$option]);
-            unset($this->required[$option]);
-            unset($this->resolved[$option]);
-            unset($this->lazy[$option]);
-            unset($this->normalizers[$option]);
-            unset($this->allowedTypes[$option]);
-            unset($this->allowedValues[$option]);
+            unset(
+                $this->defined[$option],
+                $this->defaults[$option],
+                $this->required[$option],
+                $this->resolved[$option],
+                $this->lazy[$option],
+                $this->normalizers[$option],
+                $this->allowedTypes[$option],
+                $this->allowedValues[$option]
+            );
```

Comment:
It would look ugly, as would many oneline code usages. However, Symfony CS specifies that there should be no maximum line length and oneline is prefered.